livereveal/main.js

```diff
@@ -17,8 +17,14 @@ var config_section = new configmod.ConfigSection('livereveal',
                             {base_url: utils.get_body_data("baseUrl")});
 config_section.load();
 var config = new configmod.ConfigWithDefaults(config_section, {
+    controls: true,
+    progress: true,
+    history: true,
+    width: 1140,
+    minScale: 1.0,
```

Comment:
OK, I did not think about people not showing code... the use case now makes sense... as far as it is well documented, at least with a note in the readme about the possibility to break codemirror, then I am ok to make it configurable...